rust/migrate-wicked/src/migrate.rs

```diff
@@ -43,12 +44,38 @@ fn update_parent_connection(
     Ok(())
 }
 
+fn update_bridge_ports(
+    state: &mut model::NetworkState,
+    portconfigs: HashMap<String, BridgePort>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let settings = MIGRATION_SETTINGS.get().unwrap();
+
+    for (ifc, portconfig) in portconfigs {
+        let id;
+        if let Some(c) = state.get_connection_by_interface(&ifc) {
+            id = c.id.clone();
+        } else {
+            log::warn!("Missing bridge port {} connection", ifc);
+            if !settings.continue_migration {
+                return Err(anyhow::anyhow!("Migration of {} failed", ifc).into());
```

Comment:
A bit nitpicky, because this scenario seems highly unlikely to happen, but the migration of `ifc` doesn't fail, right? It is the missing connection. The migration of the bridge failed, but because AFAICT the bridge's name isn't available here maybe something like "Migration failed because bridge is missing a port" or just "Migration of bridge failed"?

Reply:
I actually don't understand what you try to say. The migration failed while handling `ifc` and the cause is `Missing bridge port {} connection`. Isn't it the same as:

  * https://github.com/jcronenberg/agama/blob/e262cb67c1e0c0fba7080fdee5152a193a5d0c16/rust/migrate-wicked/src/migrate.rs#L94
  * https://github.com/jcronenberg/agama/blob/e262cb67c1e0c0fba7080fdee5152a193a5d0c16/rust/migrate-wicked/src/migrate.rs#L31
  
You get an Error on the failed connection and a more descriptive Warning in advance.

Reply:
Let's say you have the config of the your testcase:
br0 with ports en0 and en1
If now for some reason en1 can't be found here the bridge is missing the en1 connection, so the warning would be "Missing bridge port en1 connection" which is correct.
The error meanwhile would be "Migration of en1 failed", which isn't correct, because we don't know what happened to en1 (like I said highly unlikely because probably an error for en1 would have already occurred at that point). The migration of br0 failed.

Reply:
IMO it would be a bit more confusing to users, but if you don't agree, I'm fine with keeping it, because it seems almost impossible to occur to me.

Reply:
But what do you propose as  a better solution?